src/lustre/lustreAstHelpers.ml

```diff
@@ -71,7 +71,7 @@ let rec expr_contains_call = function
   | Ident (_, _) | ModeRef (_, _) | Const (_, _) -> false
   | RecordProject (_, e, _) | TupleProject (_, e, _) | UnaryOp (_, _, e)
   | ConvOp (_, _, e) | Quantifier (_, _, _, e) | When (_, e, _)
-  | Current (_, e) | Pre (_, e)
+  | Current (_, e) | Pre (_, e) | ChooseOp (_, _, e)
```

Comment:
This should be handled as a node call.

Reply:
I updated this particular instance, but a broader review of lustreAstHelpers is still in progress